prime-router/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml

```diff
@@ -22,16 +21,32 @@ elements:
     resource: 'Bundle.entry.resource.ofType(Patient)'
     condition: '%resource.count() = 1'
     required: true
+    constants:
+      hl7SegmentGroup: '/PATIENT_RESULT/PATIENT'
     schema: base/patient
 
+  - name: patient-note
```

Comment:
Why did things like this change when this seems to be dealing with ORU rather than A01?

Reply:
Because I am reusing the segments for the new data types. The HL7 v2 segments are the same.